static/app/views/alerts/rules/issue/details/ruleDetails.tsx

```diff
@@ -349,7 +393,7 @@ class AlertRuleDetails extends AsyncComponent<Props, State> {
   }
 }
 
-export default AlertRuleDetails;
+export default withApi(AlertRuleDetails);
```

Comment:
i think api is already available in AsyncComponents

Reply:
this.api

Reply:
looks like the api is unused here?

Reply:
good catch, i moved the api stuff i was going to do here to a different pr, will remove